frontend/components/home/HAboutUs.tsx

```
@@ -0,0 +1,118 @@
+                        <div className='about-images'>
+                            <Image src={aboutImg} alt='about img' className='w-full h-auto' />
+                            <Image src={aboutShape} alt='about shape' className='shape alltuchtopdown' />
+                            <Link href='' className='popup-video'>
+                                <svg
+                                    xmlns='http://www.w3.org/2000/svg'
+                                    width='22'
+                                    height='28'
+                                    viewBox='0 0 22 28'
+                                    fill='none'
+                                    aria-label='play video'
+                                >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)